rc/lsp.kak

```diff
@@ -1415,6 +1417,26 @@
 " | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
 }
 
+define-command -hidden lsp-breadcrumbs-request -docstring "request updating lsp_modeline_breadcrumbs for the buffer" %{
+  nop %sh{
+    window_start_line="$(echo $kak_window_range | cut -d' ' -f1)"
+
+    (printf %s "
+session  = \"${kak_session}\"
+buffile  = \"${kak_buffile}\"
+filetype = \"${kak_opt_filetype}\"
+version  = ${kak_timestamp:-0}
+method   = \"kakoune/breadcrumbs\"
+$([ -z ${kak_hook_param+x} ] || echo hook = true)
+[params]
+position_line = $kak_cursor_line
+" | eval "${kak_opt_lsp_cmd} --request") > /dev/null 2>&1 < /dev/null & }
+}
+
+hook -group lsp-breadcrumbs %arg{1} BufReload .* lsp-breadcrumbs-request
```

Comment:
we don't want to enable these at load-time (i.e. `source`)
Only when `lsp-enable{,-window}` is called, we should launch servers etc.